bot/modules/zilla.py

```diff
@@ -71,7 +71,7 @@ def bugzilla_search(self):
             except:
                 return
             for bug in res['bugs']:
-                self.controller.client.msg(self.channel, "[zilla] NEW bug: {summary} {url}{bugid}".format(summary=bug['summary'],
+                self.controller.client.msg(self.channel, "\x038\x02WARNING\x03\x02 \x032\x02NEW\x03\x02 bug: {url}{bugid} {summary}".format(summary=bug['summary'],
```

Comment:
we have to either modify upstream IRC code to integrate it in the irc module class, or to make it a separate class/lib that the module can import.
any preference?